.gitignore

```diff
@@ -0,0 +1,5 @@
+node_modules
+ORG-wdio.conf.js
+sample.png
+ios.wdio.conf.js
+android.wdio.conf.js
```

Comment:
Include all the lines from [this](https://github.com/browserstack/browserstack-examples-reusable-artifacts/blob/main/gitignores/.gitignores.js) gitignore file mentioned in our reusable artefacts repo into your gitignore file.

Reply:
done